examples/accordion/css/accordion.css

```diff
@@ -20,20 +30,33 @@
     position: relative;
     text-align: left;
     width: 100%;
+    outline: none;
 }
 
-.Accordion dt:first-child .Accordion-trigger {
-    border-radius: .3em .3em 0 0;
+.Accordion h3 {
+    margin: 0;
+    padding: 0;
+}
+
+.Accordion *:first-child .Accordion-trigger {
```

Comment:
The child elements are still colliding with the `.Accordion` border in the corners, despite this:

![screen shot 2018-09-19 at 2 58 53 pm](https://user-images.githubusercontent.com/7939037/45775178-f1678680-bc1c-11e8-982e-5724f789174d.png)

Removing the `.Accordion-trigger` child selector should fix it:

```css
.Accordion :first-child {
  border-radius: 5px 5px 0 0;
}
```